functions/src/index.ts

```diff
@@ -20,3 +21,12 @@ export const challenge = functions.https.onRequest(app_open);
 // firestore triggers
 export const custom_form_actions = typeformFunctions.custom_form_actions;
 export const typeform_confirmation = typeformFunctions.send_confirmation;
+
+// firestore compute total scripts
+export const compute_collection_totals = totalFunctions.computeCollectionTotals;
```

Comment:
For these two functions we probably don't need to export them. I'm thinking we can just write a standalone script to call them. Something like this - 

```
// initialize firebase admin with credential

import compute_collection_totals from totalFunctions.computeCollectionTotals;

compute_collection_totals();

console.log("Done ✅");
```

And then we can just call the script as `node ./computeTotals.js` and call it a day. This is mostly because this code is something that needs to just run once and after that the firestoreTriggers will handle keeping the counts up to date.

Basically we'd have a folder somewhere in this repo for just standalone "scripts" that aren't imported anywhere into the express api. We can merge this & handle turning it into a script separately since it requires initializing firebase with a service account credential file (which wouldn't be in the repo). Just documenting what needs to be done in this comment.

Reply:
@harshasrikara Just pushed these changes in the latest commit. Lmk if all looks good to merge!